.packer/win/windows-build-server.json

```diff
@@ -0,0 +1,90 @@
+{
+    "variables": {
+        "aws_access_key": "",
+        "aws_secret_key": "",
+        "instance_type": "c5.2xlarge",
+        "windows_password": ""
+    },
+    "builders": [
+        {
+            "force_deregister": true,
+            "force_delete_snapshot": true,
+            "type": "amazon-ebs",
+            "access_key": "{{user `aws_access_key`}}",
+            "secret_key": "{{user `aws_secret_key`}}",
+            "region": "eu-west-1",
+            "instance_type": "{{user `instance_type`}}",
+            "source_ami_filter": {
+                "filters": {
+                    "name": "Windows_Server-2019-English-Full-Base-*"
+                },
+                "owners": ["801119661308"],
+                "most_recent": true
+            },
+            "ami_name": "packer-jenkins-win-slave-iroha",
+            "security_group_id": "sg-0b1d8602d2597ca48",
```

Comment:
Same here.

Reply:
aws security_group_id, this group have open RDP and WinRM ports (3389/TCP, 5985 - 5986/TCP,)